.circleci/config.yml

```diff
@@ -0,0 +1,171 @@
+#
+#  Globality autogenerated CircleCI configuration
+#
+#  This file is auto generated with globality-build.
+#  You should not make any changes to this file manually
+#
+#  Any changes made to this file will be overwritten in the
+#  next version of the build.
+#
+#  See: http://github.com/globality-corp/globality-buid
```

Comment:
typo in: `http://github.com/globality-corp/globality-buid` s/buid/build/
...sorry 😞

Reply:
Indeed, but this typo has been there for 3 months: https://github.com/globality-corp/globality-build/pull/12 

We should probably fix that at some point.